pkg/controller/deadmanssnitchintegration/deadmanssnitchintegration_controller.go

```diff
@@ -35,17 +35,15 @@ var fedramp = os.Getenv("FEDRAMP") == "true"
 const (
 	deadMansSnitchAPISecretKey    = "deadmanssnitch-api-key"
 	DeadMansSnitchFinalizerPrefix = "dms.managed.openshift.io/deadmanssnitch-"
-	// These can be removed once Hive is promoted past f73ed3e in all environments
-	// Support for these conditions was removed in https://github.com/openshift/hive/pull/1604
-	legacyHivev1RunningHibernationReason  = "Running"
-	legacyHivev1ResumingHibernationReason = "Resuming"
+	// This can be removed once Hive is promoted past f73ed3e in all environments
+	// Support for this condition was removed in https://github.com/openshift/hive/pull/1604
+	legacyHivev1RunningHibernationReason = "Running"
 )
 
 var validHibernationReasons = []string{
 	hivev1.ResumingOrRunningHibernationReason,
```

Comment:
This should just go away. It leaves situations where you can still "pass" if the cluster is resuming.
```suggestion
```
(Or get rid of `validHibernationReason` entirely and just inline the check for `Running`.)